instance/config.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+
+DATABASE_URL = 'postgresql://postgres:user123*-+@localhost/flask_api'
```

Comment:
Put your enviroment variables in .env file

Reply:
Well noted. Thank you